data/settings/1.27.x/aws.toml

```diff
@@ -0,0 +1,31 @@
+[[docs.ref.config]]
```

Comment:
Can we document the behavior of this setting that changed via https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/218 in 1.27.x +?

By default for non-FIPS variants, `settings.aws.config` will default to the base64 encoded representation of 

```
[default]
use_fips_endpoint=false
```

For FIPS variants, this setting will default to the base64 encoded string of

```
[default]
use_fips_endpoint=true
```

Reply:
I won't block on this though - we can follow up with changes for this

Reply:
As discussed, these changes will be added in a separate pull request.